Source/Core/Core/PowerPC/JitILCommon/JitILBase_LoadStoreFloating.cpp

```diff
@@ -21,10 +21,23 @@ void JitILBase::lfs(UGeckoInstruction inst)
 		addr = ibuild.EmitAdd(addr, ibuild.EmitLoadGReg(inst.RA));
 
 	val = ibuild.EmitDupSingleToMReg(ibuild.EmitLoadSingle(addr));
-	ibuild.EmitStoreFReg(val, inst.RD);
-	return;
+	ibuild.EmitStoreFReg(val, inst.FD);
 }
 
+void JitILBase::lfsu(UGeckoInstruction inst)
+{
+	INSTRUCTION_START
+	JITDISABLE(bJITLoadStoreFloatingOff);
+	FALLBACK_IF(js.memcheck);
+
+	IREmitter::InstLoc addr = ibuild.EmitIntConst(inst.SIMM_16), val;
+
+	addr = ibuild.EmitAdd(addr, ibuild.EmitLoadGReg(inst.RA));
+
+	val = ibuild.EmitDupSingleToMReg(ibuild.EmitLoadSingle(addr));
```

Comment:
You can just declare this as assignment via: `IREmitter::InstLoc val = ibuild.EmitDupSingleToMReg(ibuild.EmitLoadSingle(addr));`